README.md

```diff
@@ -15,6 +15,22 @@ This adapter serves as communication interface between Iobroker and your KNX IP
 The adapter allows to generate the iobroker communication objects automatically by importing an ETS group address xml export.  
 All generated communication objects are initially configured readable and writeable, values are fetched from the knx bus on adapter restart.
 
+# Installation
```

Comment:
The recommended way to install is via adapter list or click on the github icon in the adapter. 
We should not recommend manual installation this cause problems for the most of the users

Reply:
can you add this?